middleware/isAuthorization.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {authorization : function(req,res,next){
+    if(req.decoded.role == 'admin'){
+        next()
+    }else{
+        res.status(401).json({msg:'authorization failed'})
```

Comment:
Spasinya jangan pelit pelit

```suggestion
        res.status(401).json({msg: 'authorization failed'})
```